test/terraform/openshift/oc-login.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+resource_group=$1
+cluster_name=$2
+
+apiServer=$(az aro show -g "$resource_group" -n "$cluster_name" --query apiserverProfile.url -o tsv)
+kubeUser=$(az aro list-credentials -g "$resource_group" -n "$cluster_name" | jq -r .kubeadminUsername)
+kubePassword=$(az aro list-credentials -g "$resource_group" -n "$cluster_name" | jq -r .kubeadminPassword)
+
+echo "Logging in"
+for i in {1..20}; do KUBECONFIG="$HOME/.kube/$cluster_name" oc login "$apiServer" -u "$kubeUser" -p "$kubePassword" && break; sleep 5; done
+
+echo "Creating the 'consul' project"
+# Idempotently, create and use the 'consul' project
+KUBECONFIG="$HOME/.kube/$cluster_name" oc new-project consul
+KUBECONFIG="$HOME/.kube/$cluster_name" oc project consul
+
+echo "Disabling auto-update of the worker nodes based on config changes"
```

Comment:
was this what was causing the flakes earlier?

Reply:
Yep exactly! Openshift has an operator that could reboot worker nodes if the config needs to be updated. As a result, some nodes were updating during test runs causing failures.